Content.Server/AlertLevel/AlertLevelOnPressComponent.cs

```diff
@@ -0,0 +1,18 @@
+using Content.Shared.AlertLevelOnPress;
+using Robust.Shared.Audio;
+
+namespace Content.Server.AlertLevelOnPress;
+
+[RegisterComponent]
+[Access(typeof(AlertLevelOnPressSystem))]
+public sealed partial class AlertLevelOnPressComponent : SharedAlertLevelOnPressComponent
+{
+    [DataField(required: true)]
+    public string AlertLevelOnActivate = default!;
```

Comment:
`ProtoId<AlertLevelPrototype>` instead of string and string.Empty instead of default!

Reply:
yeah so this is the only thing i can't get working, since i can't access that from shared.